Auth-MSSC/src/main/java/com/clinics/auth/ui/model/User.java

```diff
@@ -35,7 +35,7 @@
 	private Long id;
 
 	@Column(updatable = false, nullable = false)
-	private UUID uuid; //todo uuid should be null if we want edit email or pass
+	private UUID userUUID; //todo UUID should be null if we want edit email or pass <--- HE ?!?!?!
```

Comment:
Not relevant anymore ?

Reply:
Done, I was thought about auto-generated UUID by hibernate, but ....